packages/mwp-core/src/util/cookieUtils.js

```diff
@@ -11,6 +11,13 @@ export const BROWSER_ID_COOKIE = appConfig.api.isProd
 	? 'MEETUP_BROWSER_ID'
 	: 'MEETUP_BROWSER_ID_DEV';
 
+// SIFT_SESSION_ID cookie created in fastly and used by sift science to correlate spammy
+// behavior to user activity; cookie has an expiration time of 4 hours and should be
```

Comment:
this code doesn't really control the business rules (4 hour expiration, reset on logout), so in general it's better not to specify things like that separately from where they're defined - they inevitably will get out of sync. All this code really needs to know/communicate is how to parse the cookie.